account_invoice_line_description/invoice.py

```diff
@@ -43,8 +43,11 @@ def product_id_change(
                     'group_use_product_description_per_inv_line',
             ):
                 product = self.env['product.product'].browse(product)
-                if product.description:
+                description = (product.description_sale
+                               if type in ('out_invoice', 'out_refund')
+                               else product.description_purchase)
```

Comment:
No, because you can have a general description, and the super change this to [code] + product.description.